ConsoleApp/MyStack/MyStack.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp.MyStack
+{
+    internal class MyStack<T> : IEnumerable<T>
+    {
+        private Element<T> _head;
+        private int _count;
+
+        public int GetCount
```

Comment:
it should have name `Count` and instead of implementing getter & setter you can just do next
```suggestion
        public int Count { get; private set; }
```
so nobody outside your class won't change it's value but can access it